packages/core/src/components/AccordionGroup/types.ts

```diff
@@ -0,0 +1,45 @@
+import { SerializedStyles } from '@emotion/react';
+import { CommonProps } from '@global-types/emotion';
+
+export type AccordionVariant = 'empty' | 'small' | 'medium' | 'large';
+
+export type VariantStyles = {
+  [K in AccordionVariant]: SerializedStyles;
+};
+
+export interface AccordionGroupProps {
```

Comment:
We discussed a top-level prop for allowing/disallowing multiple accordions to be opened simultaneously. 
e.g. `<AccordionGroup oneOpenedOnly={true} />` -> something like that.

I don't think CTP requires that right now, so we can do it later. I will create a task for that if we decide to do it later.